routes/index.js

```diff
@@ -1,11 +1,19 @@
 const express = require('express');
 const router = express.Router();
+
+const cartRouter = require("./cartRouter")
```

Comment:
```suggestion
const express = require('express');

const cartRouter = require("./cartRouter")

const router = express.Router();
```
필요한 모듈을 불러온 뒤, 이를 활용하는 코드를 작성하는 순서가 일반적입니다.